client/lib/analytics/index.js

```diff
@@ -125,6 +125,11 @@ var analytics = {
 		}
 	},
 
+	recordEvent: function( tracksEventName, tracksEventProperties, category, action, label, value ) {
```

Comment:
We can, but if you change the names for GA now, I think we lose the ability to track the history before the change. To answer the earlier question, Ideally we can send to both as a redundancy and sanity checker.